### ISSUE TITLE
Biquad Optimizer 2.0

### ISSUE THREAD

**Issue by jaakkopasanen** [jaakkopasanen/AutoEq#91]
- Investigate if Tensorflow 2's eager execution mode would reduce the initialization time which currently takes a big chunk of the optimization time.
- Investigate if different learning rate adjustment strategy would make the optimization converge faster.
- Test out other optimization libraries, ones which would work natively with Numpy.
- Test graphic eq style filter initializations. Current heuristics are not probably necessary.
- Move unused filters to next iteration.
- Train neural network which predicts 5+5 peaking filters in one forward pass
- Train neural network which predicts 10, 15 and 31 fixed filters in one forward pass
- There networks could be combined to have 66 filters output

**Comment by bhack** [comment]
Any news on this?
